site/content/installation/installing-ngf/manifests.md

```diff
@@ -19,6 +19,10 @@ To complete this guide, you'll need to install:
 
 Deploying NGINX Gateway Fabric with Kubernetes manifests takes only a few steps. With manifests, you can configure your deployment exactly how you want. Manifests also make it easy to replicate deployments across environments or clusters, ensuring consistency.
 
+- If you’d like to use NGINX Plus:
+  - To pull from the F5 Container registry, configure a docker registry secret using your JWT token from the MyF5 portal by following the instructions from [here](https://docs.nginx.com/nginx-gateway-fabric/installation/ngf-images/jwt-token-docker-secret). Make sure to specify the secret using `nginxGateway.serviceAccount.imagePullSecretName` or `nginxGateway.serviceAccount.imagePullSecretsNames` parameter.
+  - To generate manifests for NGINX Plus, use `make generate-manifests-plus` at the root directory of the repository. You should see `image` field updated with `nginx-gateway-fabric/nginx-plus:edge`.
```

Comment:
You can still mention the alternative option of pulling the image and pushing to a private registry, like you do in the helm guide.

Regarding generating the manifests, since we are committing them, a user shouldn't need to do that. Instead, you can just mention that they should update the image field in the manifest.

Reply:
You should mention in the installation steps below how to install using the plus manifests instead of the regular manifests, similar to how the helm guide has two sets of instructions.